README.md

```diff
@@ -17,7 +17,7 @@ Project Status:
 | [Diagnostics](https://greenelab.github.io/covid19-review/#diagnostics) | Molecular and Serologic Diagnostic Technologies for SARS-CoV-2| ~#818~ #1094 | Revision in progress | _mSystems_ | [Release](https://github.com/greenelab/covid19-review/releases/tag/diagnostics-v1); [Preprint](https://arxiv.org/abs/2204.12598) |
 | [Pharmaceuticals](https://greenelab.github.io/covid19-review/#identification-and-development-of-therapeutics-for-covid-19) | Identification and Development of Therapeutics for COVID-19 | ✔️ | [Published](https://doi.org/10.1128/mSystems.00233-21) 🎉 | _mSystems_ | [Release](https://github.com/greenelab/covid19-review/releases/tag/pharmaceuticals-v3); [Preprint](http://arxiv.org/abs/2103.02723) |
 | [Vaccines- Established Platforms](https://greenelab.github.io/covid19-review/#application-of-traditional-vaccine-development-strategies-to-sars-cov-2) | Application of Traditional Vaccine Development Strategies to SARS-CoV-2 | #830 | Submitted | Submitted to _mSystems_ | [Release](https://github.com/greenelab/covid19-review/releases/tag/vaccines-trad-v1); [Preprint](https://arxiv.org/abs/2208.08907) |
-| [Vaccines- Novel Platforms](https://greenelab.github.io/covid19-review/#the-coming-of-age-of-nucleic-acid-vaccines-during-covid-19) | The Coming of Age of Nucleic Acid Vaccines during COVID-19 | #830 | Submitted; preprint pending | Submitted to _mSystems_ | [Release](https://github.com/greenelab/covid19-review/releases/tag/vaccines-novel-v1) |
+| [Vaccines- Novel Platforms](https://greenelab.github.io/covid19-review/#the-coming-of-age-of-nucleic-acid-vaccines-during-covid-19) | The Coming of Age of Nucleic Acid Vaccines during COVID-19 | #830 | Accepted 🎉 | _mSystems_ | [Release](https://github.com/greenelab/covid19-review/releases/tag/vaccines-novel-v1) |
```

Comment:
```suggestion
| [Vaccines- Novel Platforms](https://greenelab.github.io/covid19-review/#the-coming-of-age-of-nucleic-acid-vaccines-during-covid-19) | The Coming of Age of Nucleic Acid Vaccines during COVID-19 | #830 | Accepted 🎉 | _mSystems_ | [Release](https://github.com/greenelab/covid19-review/releases/tag/vaccines-novel-v2 |
```
Pre-emptively updating the tag, because I'll create this release as soon as we merge this.